guide/english/javascript/arrow-functions/index.md

```diff
@@ -77,7 +77,14 @@ function Person(){
 
 var p = new Person();
 ```
+An arrow function doesn't go well with function arguments. For example, if you do not know the number of arguments to a function you will be using in a function you do not leave it empty, rather you can use spread operator:
```

Comment:
I believe this is the [`rest`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) operator.  

Also, please update, and clarify what it means when you say
> doesn't go well with function arguments

This sentence could be better worded.